src/OrderFormAcceptanceTests.Steps/Steps/AssociatedServices.cs

```diff
@@ -229,6 +229,16 @@ public void GivenThereIsNoAssociatedServiceInTheOrderButTheSectionIsComplete()
             SetOrderAssociatedServicesSectionToComplete();
         }
 
+        [Given(@"the supplier added to the order has an associated service with a declarative flat price")]
+        public void GivenTheSupplierAddedToTheOrderHasAnAssociatedServiceWithADeclarativeFlatPrice()
+        {
+            var order = (Order)Context["CreatedOrder"];
+            order.SupplierId = 100000;
+            order.SupplierName = "Really Kool Corporation";
+            order.Update(Test.ConnectionString);
```

Comment:
Would it be worth finding this via the db's rather than hardcoded?

Reply:
without a doubt, that'd be nicer, was trying to follow the pattern of the other tests though, that'd be tech debt I think

Reply:
I'll be tackling that tech debt when the latest round of AC tests are complete, but getting a start on it with this PR would be handy

Reply:
I think if I got a start on it with this PR, it'd essentially be doing the entire tech debt, and that seems like it should be its own task

Reply:
If I was to vote, I'd say I'd rather callum just followed what we/I've done, and you/I tackled this so to keep callum available for dev work. If he or any dev starts to refractor they'll never make it out this side of the August deadline

Reply:
righto, I'll approve the PR